expression/builtin_op.go

```diff
@@ -339,13 +339,16 @@ func (c *isTrueOrFalseFunctionClass) getFunction(ctx context.Context, args []Exp
 		return nil, errors.Trace(err)
 	}
 
-	argTp := tpInt
```

Comment:
this is an equivalent replacement?
should we keep it here, and use `fieldTp2EvalTp` in line 343?

Reply:
yes, `ClassReal` == `tpReal` and `ClassDecimal` == `tpDecimal`